packages/paste-website/src/pages/blog/2024-08-23-paste-newsletter.mdx

```diff
@@ -0,0 +1,189 @@
+export const meta = {
+  title: "Paste Design System Newsletter - August 2024 Edition",
+  slug: "/blog/2024-08-23-paste-newsletter/",
+  date: "2024-08-23",
+  author: "Kristian Antrobus",
```

Comment:
missed this on the last one, we typically have done "Paste" as the author for the newsletters, but totally up to you! just so you know you don't _have_ to put your name up there if you don't want to lol

Reply:
Na I'll keep it as Paste haha. If anyones name deserves to be there is Sarahs 🤣